source/src/plugins/MeanWithinExpectedTest.cc

```diff
@@ -60,6 +62,10 @@ namespace dqm4hep {
       float m_expectedMean;
       float m_meanDeviationLower;
       float m_meanDeviationUpper;
+      float m_percentage;
+
+      std::vector<std::string> m_testTypes;
```

Comment:
Rename it to `m_properties`

Reply:
Do you mean m_percentage or m_testTypes?

Reply:
For m_testTypes